MainProject/UABCompany.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace MainProject
+{
+    public class UABCompany : Company
+    {
+        private string ownerName;
```

Comment:
Naming rules: private field in class name starts with "_" and lower case letter (e.g. _ownerName)
Please fix